File: checkpoint0/src/main/java/com/springernature/checkpoint0/GameRenderer.java
package com.springernature.checkpoint0;

public class GameRenderer {
    private Game game;

    GameRenderer(Game game) {
        this.game = game;
    }

    String render() {
        String message;
        if (game.isGameADraw()) {
            message = "Game is a draw!";
        } else {
            message = "\n" +
                    new BoardRenderer(game.board()).render() + "\n" +
                    "Player " + game.nextCellToPlace() + " please make a move: ";
        }
        return message;
    }
}

File: checkpoint1/src/main/kotlin/com/springernature/checkpoint1/GameRenderer.kt
package com.springernature.checkpoint1

class GameRenderer(private val game: Game) {

    fun render(): String {
        val message =
            if (game.isGameADraw) {
                "Game is a draw!"
            } else {
                "\n" + BoardRenderer(game.board()).render() + "\n" +
                    "Player " + game.nextCellToPlace() + " please make a move: "
            }
        return message
    }
}